rest_framework/schemas.py

```diff
@@ -467,31 +466,36 @@ def get_description(self, path, method, view):
         This will be based on the method docstring if one exists,
         or else the class docstring.
         """
-        method_name = getattr(view, 'action', method.lower())
-        method_docstring = getattr(view, method_name, None).__doc__
-        if method_docstring:
+        method = method.lower()
+        method_name = getattr(view, 'action', method)
+        view_method = getattr(view, method_name, None)
+        description = None
+        if view_method:
             # An explicit docstring on the method or action.
-            return formatting.dedent(smart_text(method_docstring))
-
-        description = view.get_view_description()
+            description = api_settings.VIEW_DESCRIPTION_FUNCTION(view_method)
+        if not description:
```

Comment:
It's not `else` because the condition is `if view_method` and - as I get the idea - the logic was to fall back to view class' `get_view_description()` method if method-specific docstring is not available.

It could be else with `if view_method.__doc__`, but then it would update logic of `VIEW_DESCRIPTION_FUNCTION` (`views.get_view_description`) and prevent description functions that take other sources than docstrings (e.g. if someone decides to generate code from annotations or something).

One thing I see is when docstring has only one section (e.g. `"""read: Retrieves data"""`), but route decorator allows other methods, and it happens that `method_name` is, e.g. `POST`. The code below would fail to find `sections["create"]` but we'll never fall back to the view class' docstring. This may or may not be an issue - I'm not sure what would be the correct behavior - to fall back or to return an empty string. Thought I'd point this out.